compiler/AST/primitive.cpp

```diff
@@ -679,6 +679,9 @@ initPrimitive() {
   // These are used for task-aware allocation.
   prim_def(PRIM_SIZEOF, "sizeof", returnInfoSizeType);
```

Comment:
Let's rename this one to PRIM_SIZEOF_BUNDLE since it's only used in computing the size of the argument bundle for tasks etc.